src/universal/components/TeamInvitationGoogleCreateAccount.tsx

```diff
@@ -99,21 +100,32 @@ class TeamInvitationGoogleCreateAccount extends Component<Props, State> {
   render () {
     const {error, submitting, verifiedInvitation} = this.props
     const {isEmailFallback} = this.state
-    const {teamInvitation, teamName} = verifiedInvitation
+    const {meetingType, teamInvitation, teamName} = verifiedInvitation
     if (!teamInvitation) return null
     const {email} = teamInvitation
     return (
       <StyledDialog>
         <Helmet title={`Sign up with Google | Team Invitation`} />
-        <DialogTitle>Welcome!</DialogTitle>
+        <DialogTitle>
+          {meetingType
+            ? `Join ${meetingTypeToLabel[meetingType]} Meeting in Progress`
+            : 'Join Team'}
+        </DialogTitle>
         <StyledContent>
           <CopyMargins>
             <InvitationDialogCopy>
               It looks like your email is hosted by Google.
             </InvitationDialogCopy>
-            <InvitationDialogCopy>
-              Tap below for immediate access to your team: <TeamName>{teamName}</TeamName>
-            </InvitationDialogCopy>
+            {meetingType ? (
+              <InvitationDialogCopy>
+                You’re just 1 step away from participating with your team:{' '}
```

Comment:
same as above, I'm not sure what "1 step" means. "tap below for immediate access" seems clearer to me

can also start conditional here